codegen/runner/runner.go

```diff
@@ -59,6 +59,23 @@ func main() {
 		err, fmt.Sprintf("cant get abs path of config dir %s", configDirName),
 	)
 
+	packageHelper, err := codegen.NewPackageHelper(
```

Comment:
I imagine that gatewaySpec goes away when the module system fully generates the required code. Open to suggestions about how to share the packageHelper but it doesn't seem unusual to have 2 while we have 2 separate build systems working together.